wear/src/main/java/io/homeassistant/companion/android/util/CommonFunctions.kt

```diff
@@ -74,7 +124,13 @@ fun getIcon(entity: Entity<Map<String, Any>>?, domain: String, context: Context)
             } else { // For SimplifiedEntity without state, use a more generic icon
                 CommunityMaterial.Icon2.cmd_light_switch
             }
-            else -> CommunityMaterial.Icon.cmd_cellphone
+            "timer" -> CommunityMaterial.Icon3.cmd_timer_outline
+            "updater" -> CommunityMaterial.Icon.cmd_cloud_upload
+            "vacuum" -> CommunityMaterial.Icon3.cmd_robot_vacuum
+            "water_heater" -> CommunityMaterial.Icon3.cmd_thermometer
+            "weather" -> CommunityMaterial.Icon3.cmd_weather_cloudy
+            "zone" -> CommunityMaterial.Icon3.cmd_map_marker_radius
+            else -> CommunityMaterial.Icon.cmd_bookmark

```

Comment:
For consistency I'd say the `CommunityMaterial.Icon.cmd_cellphone`s elsewhere also should be updated.